Backend/app/rider-platform/rider-app/Main/src/Beckn/ACL/OnUpdate.hs

```diff
@@ -142,6 +142,8 @@ parseEvent _ (OnUpdate.RideAssigned taEvent) = do
 parseEvent _ (OnUpdate.RideStarted rsEvent) = do
   let personTagsGroup = rsEvent.fulfillment.person.tags
   let tripStartLocation = getLocationFromTag personTagsGroup "current_location" "current_location_lat" "current_location_lon"
+  authorization <- fromMaybeM (InvalidRequest "authorization is not present in RideStarted Event.") $ rsEvent.fulfillment.start.authorization
```

Comment:
What if authorization is not there in case of normal booking? Is it backward compatible?

Reply:
I checked in mkFulfillment and mkFulfillmentV2. Authorization is present in both. About compatibility, I send startRideOtp when endRideOtp is not there at BPP. Can you review that logic once?